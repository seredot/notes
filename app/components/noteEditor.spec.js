import "react-native";
import React from "react";
import NoteEditor from "./noteEditor";

import renderer from "react-test-renderer";

describe("Testing NoteEditor component", () => {
	it("renders correctly", () => {
		const tree = renderer.create(<NoteEditor />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
