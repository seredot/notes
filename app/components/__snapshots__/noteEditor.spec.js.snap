// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing NoteEditor component renders correctly 1`] = `
<View
  style={
    Object {
      "alignSelf": "stretch",
      "flex": 1,
      "paddingVertical": 6,
    }
  }
>
  <TextInput
    allowFontScaling={true}
    multiline={true}
    style={
      Object {
        "alignSelf": "stretch",
        "backgroundColor": "#fff",
        "color": "#333",
        "flex": 1,
        "fontSize": 16,
        "paddingHorizontal": 12,
      }
    }
    underlineColorAndroid="transparent"
  />
</View>
`;
